examples/experimental/opensuse-tumbleweed.yaml

```diff
@@ -17,6 +17,8 @@ images:
 - location: "https://provo-mirror.opensuse.org/ports/aarch64/tumbleweed/appliances/openSUSE-Tumbleweed-ARM-JeOS-efi.aarch64.qcow2"
   arch: "aarch64"
 
+# The mount type is set to reverse-sshfs because 9p (default for QEMU) is not supported by the guest kernel. VZ users should specify `--mount-type=virtiofs` on `limactl create` for better performance.
+mountType: "reverse-sshfs"
```

Comment:
I wonder if we shouldn't have some builtin way to specify this, as an identical change is being made to about 10 templates. They will all run with `reverse-sshfs` when being used with `vz` (the new default in 1.0) unless the user explicitly overrides it.

I wish I could think about something better than this:

```yaml
# mountType will be reverse-sshfs when using QEMU or virtiofs when using VZ
mountType: not-9p
```

Reply:
What about
```yam
mountType: "default"
mountTypeUnsupported: ["9p"]
```

Reply:
Yes, that is better and more flexible. But since it takes a list it should be plural: 

```diff
  mountType: "default"
- mountTypeUnsupported: ["9p"]
+ mountTypesUnsupported: ["9p"]
```

Reply:
done